rosetta-wallet/src/main.rs

```diff
@@ -79,6 +92,33 @@ async fn main() -> Result<()> {
             let balance = wallet.balance().await?;
             println!("{}", rosetta_client::amount_to_string(&balance)?);
         }
+        Command::DeployContract(DeployContractOpts { bin_path }) => {
+            match wallet.config().blockchain {
+                "bitcoin" => {
+                    anyhow::bail!("Not implemented");
+                }
+                _ => {
```

Comment:
would probably be nicer if we'd compile the smart contract here too. We already have some code to do that in tests.